### PR TITLE
[deploy] Add liveness and readiness probes

### PR DESCRIPTION
**Pull request by awgreene**
Updated main.go to include an http endpoint that returns a 200 status when pinged for use in readiness and liveness probes.
Updated the operator.yaml to include a readiness and liveness probe.

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -19,9 +19,19 @@ spec:
           ports:
           - containerPort: 60000
             name: metrics
+          - containerPort: 8080
+            name: healthz
           command:
           - marketplace-operator
           imagePullPolicy: Always
+          livenessProbe:
```
Comment (aravindhp) [inline deploy/operator.yaml]: Please add this to the CSV also.

Comment (awgreene) [reply deploy/operator.yaml]: Added to the CSV.

**Comment by awgreene** [comment]
Relies on changes introduced in PR #49

**Comment by kevinrizza** [comment]
Probably outside the scope of this pr, but I think we should start to consider doing something more than have a single long main() method. Right now we have framework logic around creating the manager and the controllers in this method, which seems somewhat orthogonal to adding an http endpoint to the process.

/lgtm

**Pull request merged**